config/config.go

```diff
@@ -84,3 +86,16 @@ func (c *Config) SetIgnoreRule(ignoreRule string) {
 		c.IgnoreRule[r] = true
 	}
 }
+
+func (c *Config) SetVarfile(varfile string) {
+	// Automatically, `terraform.tfvars` loaded
+	c.Varfile = append(c.Varfile, "terraform.tfvars")
+
+	if varfile == "" {
+		return
+	}
+	varfiles := strings.Split(varfile, ",")
+	for _, v := range varfiles {
```

Comment:
__[gometalinter]__
* [gosimple] should replace loop with c.Varfile = append(c.Varfile, varfiles...)

<!-- Comment by [SideCI](https://sideci.com) -->